lib/temporal.rb

```diff
@@ -81,6 +84,54 @@ def signal_workflow(workflow, signal, workflow_id, run_id, input = nil)
       )
     end
 
+    # Long polls for a workflow to be completed and returns whatever the execute function
+    # returned.
+    # run_id of nil: await the entire workflow completion.  This can span multiple runs
+    # in the case where the workflow uses continue as new.
```

Comment:
Let's add a note that this is an unstable API and we might change it to give us some flexibility